src/App.tsx

```diff
@@ -1,24 +1,182 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useEffect,
+  useState,
+  ChangeEvent,
+  FormEvent,
+} from 'react';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { ErrorPanel } from './components/ErrorPanel';
+
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+
+import { getTodos, addTodo, removeTodo } from './api/todos';
+
+const USER_ID = 6160;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [isEditing] = useState(false);
```

Comment:
seems this variable is always falsy in your code. Do you really need it? What for?

Reply:
its created for later usage, forgot to remove from the 2nd part

Reply:
oh, kk. You can just create a separate boolean variable like `shouldRenderThirdPartOfTheTask` to make it clear to the reviewers. Or to write a comment in the PR description

Reply:
yes, agree